routemaster/db/model.py

```diff
@@ -10,12 +13,32 @@
     DateTime,
     MetaData,
     ForeignKey,
+    FetchedValue,
     ForeignKeyConstraint,
 )
 from sqlalchemy.dialects.postgresql import JSONB
 
 metadata = MetaData()
 
+sync_label_updated_column = DDL(
+    '''
+    CREATE OR REPLACE FUNCTION sync_label_updated_column_fn()
+        RETURNS TRIGGER AS
+            $$
+                BEGIN
+                    NEW.updated = now();
+                    RETURN NEW;
+                END;
+            $$
+        LANGUAGE PLPGSQL;
```

Comment:
We should document that we're explicitly tied to Postgres if we haven't already done so.

Reply:
3118376a03c7cea46150e33fc828afd3a80c1c71